Mathlib/Algebra/Star/Basic.lean

```diff
@@ -18,14 +18,14 @@ We introduce the basic algebraic notions of star monoids, star rings, and star m
 A star algebra is simply a star ring that is also a star module.
 
 These are implemented as "mixin" typeclasses, so to summon a star ring (for example)
-one needs to write `(R : Type*) [Ring R] [StarRing R]`.
+one needs to write `(R : Type*) [NonUnitalNonAssocSemiring R] [StarRing R]`.
```

Comment:
```suggestion
one needs to write `(R : Type*) [Ring R] [StarRing R]`.
```
since the text above is about "star ring"s not the non-untal non-associative case.